core/src/main/java/module-info.java

```diff
@@ -0,0 +1,53 @@
+import org.jdbi.v3.core.spi.JdbiPlugin;
+
+module org.jdbi.v3.core {
+
+	exports org.jdbi.v3.core;
+	exports org.jdbi.v3.core.annotation;
+	exports org.jdbi.v3.core.annotation.internal;
+	exports org.jdbi.v3.core.argument;
+	exports org.jdbi.v3.core.argument.internal;
+	exports org.jdbi.v3.core.argument.internal.strategies;
+	exports org.jdbi.v3.core.array;
+	exports org.jdbi.v3.core.async;
+	exports org.jdbi.v3.core.cache;
+	exports org.jdbi.v3.core.cache.internal;
+	exports org.jdbi.v3.core.codec;
+	exports org.jdbi.v3.core.collector;
+	exports org.jdbi.v3.core.config;
+	exports org.jdbi.v3.core.config.internal;
+	exports org.jdbi.v3.core.enums;
+	exports org.jdbi.v3.core.enums.internal;
+	exports org.jdbi.v3.core.extension;
+	exports org.jdbi.v3.core.extension.annotation;
+	exports org.jdbi.v3.core.generic;
+	exports org.jdbi.v3.core.h2;
+	exports org.jdbi.v3.core.interceptor;
+	exports org.jdbi.v3.core.internal;
```

Comment:
Should we be exporting `.internal` packages?

Reply:
I just opened everything, you can change if you want

Reply:
Unless there's a reason to keep it, let's not export `internal` packages. All the rest I think are fine to export.